mscore/resourceManager.cpp

```diff
@@ -185,6 +186,8 @@ void ResourceManager::download()
             if (result) {
                   QFile::remove(localPath);
                   button->setText(tr("Updated"));
+                  //  retranslate the UI
+                  setMscoreLocale(localeName);
```

Comment:
When guarded by `if (data == buttonMap.first())` this would only be done when really needed, i.e. if the current active language got updated